nevergrad/functions/pbo/core.py

```diff
@@ -34,27 +35,25 @@ class PBOFunction(base.ExperimentFunction):
     def __init__(self, fid: int = 1, iid: int = 0, dim: int = 16, instrumentation: str = "Softmax") -> None:
         if fid in [21, 23]:
             assert np.sqrt(dim).is_integer(), "Dimension needs to be a perfect square for the selected problem"
-        self.f_internal = IOH_function(fid = fid, dim = dim, iid = iid, suite = "PBO")
-        assert instrumentation in ["Softmax", "Unordered"], "The only valid options for 'instrumentation' are 'Softmax' and 'Unordered'"
+        self.f_internal = IOH_function(fid=fid, dim=dim, iid=iid, suite="PBO")
+        assert instrumentation in ["Softmax", "Ordered"], "The only valid options for 'instrumentation' are 'Softmax' and 'Ordered'"
```

Comment:
may be more acurate to say "ordered" or "unordered"
or possibly just use a boolean